docs/source/using/profiles.adoc

```diff
@@ -31,10 +31,11 @@ There are two ways to create a new profile.
 
 [NOTE]
 ====
-Profile name can only consist of alphanumeric characters and *-* as a separator, for example:
+Profile name can only consist of alphanumeric characters.
+The use of underscore *_* and *-* as a separator is allowd. For example:
 
-- foo, bar123, demo-profile and 1stProfile are considered as valid profile names.
-- $foo, bar#@, profile@123 are invalid profile names.
+- myprofile, myprofile_123, demo-profile and 1stProfile are considered as valid profile names.
+- $myprofile, myprofile#@, profile@123, -myprofile are invalid profile names.
```

Comment:
By the docs it is not clear why '-myprofile' should be disallowed.

Reply:
it should be described that this can be interpreted as flag/option (and also might be disallowed by the underlying hypervisor tools). However, do we really need to explain the technical details?

Reply:
IMO `-` and `_` should be only allowed as separator.  @hferentschik why do you think `-myprofile` is a valid profile name?

Reply:
I think we can remove the line about invalid profile names. As it adds to the confusion that why these are invalid names.

Reply:
TBH, I think we can remove the whole example section.